internal/setup/setup.go

```diff
@@ -623,3 +650,75 @@ func Select(release *Release, slices []SliceKey) (*Selection, error) {
 
 	return selection, nil
 }
+
+// convertPathInfoToYamlPath converts a PathInfo object to a yamlPath object.
+func convertPathInfoToYamlPath(pi *PathInfo) (*yamlPath, error) {
+	if pi == nil {
+		return nil, nil
+	}
+	path := &yamlPath{
+		Mode:    yamlMode(pi.Mode),
+		Mutable: pi.Mutable,
+		Until:   pi.Until,
+		Arch:    yamlArch{List: pi.Arch},
+	}
+	switch pi.Kind {
+	case DirPath:
+		path.Dir = true
+	case CopyPath:
+		path.Copy = pi.Info
+	case TextPath:
+		path.Text = &pi.Info
+	case SymlinkPath:
+		path.Symlink = pi.Info
+	case GlobPath:
```

Comment:
I guess this kind is intentionally not supported?

Reply:
Yup, this is intentionally left empty. Nothing more needs to be done for this type.

Reply:
A comment there saying so wouldn't hurt.